src/renderers/WebGLRenderer.js

```diff
@@ -1526,7 +1526,7 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 		// update scene graph
 
-		if ( scene.autoUpdate === true ) scene.updateMatrixWorld();
+		if ( scene.autoUpdate === true ) scene.updateMatrixWorld( );
```

Comment:
Hmmm. I would have expected you would only call `updateMatrixWorld()` here if you wanted to traverse the entire tree and update all transforms for efficiency or performance reasons... If so, you need to force children to be updated.

If you only want to update the scene's matrix, then we are in trouble, because there is breakage if a user transforms the scene -- for example, the helpers will not work.